go/node/deployment/v1/errors.go

```diff
@@ -32,48 +33,49 @@ const (
+	ErrDeploymentExists = cerrors.RegisterWithGRPCCode(ModuleName, errDeploymentExists, codes.AlreadyExists, "Deployment exists")
 	// ErrDeploymentNotFound is the error when deployment not found
-	ErrDeploymentNotFound = cerrors.Register(ModuleName, errDeploymentNotFound, "Deployment not found")
+	ErrDeploymentNotFound = cerrors.RegisterWithGRPCCode(ModuleName, errDeploymentNotFound, codes.NotFound, "Deployment not found")
 	// ErrDeploymentClosed is the error when deployment is closed
-	ErrDeploymentClosed = cerrors.Register(ModuleName, errDeploymentClosed, "Deployment closed")
+	ErrDeploymentClosed = cerrors.RegisterWithGRPCCode(ModuleName, errDeploymentClosed, codes.FailedPrecondition, "Deployment closed")
 	// ErrOwnerAcctMissing is the error for owner account missing
-	ErrOwnerAcctMissing = cerrors.Register(ModuleName, errOwnerAcctMissing, "Owner account missing")
+	ErrOwnerAcctMissing = cerrors.RegisterWithGRPCCode(ModuleName, errOwnerAcctMissing, codes.InvalidArgument, "Owner account missing")
 	// ErrInvalidGroups is the error when groups are empty
-	ErrInvalidGroups = cerrors.Register(ModuleName, errInvalidGroups, "Invalid groups")
+	ErrInvalidGroups = cerrors.RegisterWithGRPCCode(ModuleName, errInvalidGroups, codes.InvalidArgument, "Invalid groups")
 	// ErrInvalidDeploymentID is the error for invalid deployment id
-	ErrInvalidDeploymentID = cerrors.Register(ModuleName, errInvalidDeploymentID, "Invalid: deployment id")
+	ErrInvalidDeploymentID = cerrors.RegisterWithGRPCCode(ModuleName, errInvalidDeploymentID, codes.InvalidArgument, "Invalid: deployment id")
 	// ErrEmptyHash is the error when version is empty
-	ErrEmptyHash = cerrors.Register(ModuleName, errEmptyHash, "Invalid: empty hash")
+	ErrEmptyHash = cerrors.RegisterWithGRPCCode(ModuleName, errEmptyHash, codes.InvalidArgument, "Invalid: empty hash")
 	// ErrInvalidHash is the error when version is invalid
-	ErrInvalidHash = cerrors.Register(ModuleName, errInvalidHash, "Invalid: deployment hash")
+	ErrInvalidHash = cerrors.RegisterWithGRPCCode(ModuleName, errInvalidHash, codes.InvalidArgument, "Invalid: deployment hash")
 	// ErrInternal is the error for internal error
-	ErrInternal = cerrors.Register(ModuleName, errInternal, "internal error")
+	ErrInternal = cerrors.RegisterWithGRPCCode(ModuleName, errInternal, codes.Internal, "internal error")
 	// ErrInvalidDeployment = is the error when deployment does not pass validation
-	ErrInvalidDeployment = cerrors.Register(ModuleName, errInvalidDeployment, "Invalid deployment")
+	ErrInvalidDeployment = cerrors.RegisterWithGRPCCode(ModuleName, errInvalidDeployment, codes.InvalidArgument, "Invalid deployment")
 	// ErrInvalidGroupID is the error when already deployment exists
-	ErrInvalidGroupID = cerrors.Register(ModuleName, errInvalidGroupID, "Deployment exists")
+	ErrInvalidGroupID = cerrors.RegisterWithGRPCCode(ModuleName, errInvalidGroupID, codes.InvalidArgument, "Deployment exists")
```

Comment:
I am not sure about this error mapping - it can be both `codes.InvalidArgument` and `codes.AlreadyExists`.